SeanXYTan/ACIT2911
# Merge qtw2 into dev for deliverable two

- Added RecipieLogo.png to logo branch
- Added NewRecipieLogo and PieLogo to logo branch
- Add status bar funcionality to gui Update Qt project to reflect week 1 and deliverables
- Update UI further in preparation of building logic functions
- Resize logo to be sqaure in small and large
- Remove orphaned imports and functions from revent/recipie
- Add docstrings and typing to functions in revent
- Update build_qtui.ps1 to directly build qtui.py Fix missing type declaration from revent.py Regen qtui with minor changes from creator
- Refactor font choices and squish together UI
- Add basic keyboard shortcut functionality
- Add various event captures and functions related to ui search
- Clean up code and fix minor bug in revent search events
- Create unit test for exact match function
- Add beginning of print functionality
- Fixed one of the asserts and seperated the tests
- Made test_exact_search.py easier
- Fixed an assert that checks the list for Recipe instances, when none were made
- Adjusted docstrings to the appropriate documentation
- Add working print functionality
- Refactored test with an additional check for user input capitalization
- Fix update_qtext_print via use of regex
- Fix regex and formatting for print display
- Add skeleton function to display search results
- Modified test_exact_search.py (changed import statement and a function name) & Added exact_search function to esearch.py
- Added additional tests and 2 additional things in the RECIPE_LIST to test the new function
- Add function to display recipe from results on click
- Add ability to display recipe in new window
- Fix multiwin variable name to be more clear
- Added additional code for exact_function to handle singular and plural words comparison
- Setup exact search function in revent
- Maybe fix search functionality
- Fix search functionality
- Fix ingredients/directions labels
- Fix documentation on set_md_recipe method
- Begin to add multithreading
- Add multithreaded search capability and worker setup
- Add functionality to lock ui when searching
- Add quote functionality, rework UI slightly
